### PR TITLE
Namadillo quick fixes

### PR DESCRIPTION
**Pull request by emccorson**
Fixes from the review of #708.

### CODE REVIEW

File: apps/namadillo/src/App/Governance/LiveGovernanceProposals.tsx [hunk 1]
```diff
@@ -106,14 +106,18 @@ export const LiveGovernanceProposals: React.FC<LiveGovernanceProposalsProps> = (
...
             props.isExtensionConnected ?
               props.votedProposalIds.includes(proposal.id)
             : undefined;
 
           return (
-            <ProposalListItem proposal={proposal} voted={voted} key={index} />
+            <ProposalListItem
+              proposal={proposal}
+              voted={voted}
+              key={proposal.id.toString()}
```
Comment (pedrorezende) [inline apps/namadillo/src/App/Governance/LiveGovernanceProposals.tsx:119]: Could we be more specific on this key, so we don't have a duplicate key if we display this proposal somewhere else?

File: apps/namadillo/src/App/Governance/UpcomingProposals.tsx [hunk 2]
```diff
@@ -59,8 +59,8 @@ export const UpcomingProposals: React.FC<{
         as="ul"
         className="dark-scrollbar overscroll-contain overflow-x-auto"
       >
-        {upcomingProposals.map((proposal, index) => (
-          <ProposalListItem proposal={proposal} key={index} />
+        {upcomingProposals.map((proposal) => (
+          <ProposalListItem proposal={proposal} key={proposal.id.toString()} />
```
Comment (pedrorezende) [inline apps/namadillo/src/App/Governance/UpcomingProposals.tsx:63]: Same here

**Review by pedrorezende — CHANGES_REQUESTED** [review]
Good one! :) Just a minor comment!

**Review by pedrorezende — APPROVED** [review]
LGTM!

**Review by jurevans — APPROVED** [review]
LGTM!

**Pull request merged**